app/components/primary_navigation_component.html.erb

```diff
@@ -2,13 +2,13 @@
   <div class="govuk-width-container">
     <ul class="app-primary-navigation__list">
       <% items.each do |item| %>
-        <% if highlighted_tab?(item, request.fullpath) %>
+        <% if item.current %>
```

Comment:
is this meant to be `item.current?`

Reply:
The failing tests suggest `.current` doesn't work?